frontend/react/src/pages/Auth/presentation/Login.jsx

```diff
@@ -54,11 +54,20 @@ function Login() {
     }
   }, []);
 
+  // Navigate to home page when logo icon is clicked
+  const handleLogoClick = () => {
+    navigate("/");
+  };
+
   return (
     <>
       <section className="float-right relative h-screen w-screen lg:w-[40%]">
-        {/* Top Left Icon and Text */}
-        <div className="absolute right-3 top-3">
+        {/* Top Right Icon and Text */}
+        <div 
+          className="absolute right-3 top-3" 
+          onClick={handleLogoClick} 
+          style={{ cursor: "pointer", zIndex: 1 }}
```

Comment:
Could you directly incorporate this into the `className` attribute, using TailwindCSS(the project is already using TailwindCSS).